src/build-dep-graph.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const semver = require('semver');
 const dependencyTypes = require('./dependency-types');
 const readJson = require('./json').read;
-const getWorkspacesPaths = require('./get-workspaces-paths');
+const getWorkspaces = require('./get-workspaces-paths').getWorkspacesPaths;
```

Comment:
`const { getWorkspacesPaths } = require('./get-workspaces-paths');` ?

Reply:
I dropped commit to refactor code in src/build-dep-graph.js, I will do that in other PR